compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -951,6 +951,20 @@ static bool hasUnfusableUseInDispatch(Value v, Operation *dispatchOp) {
         return true;
       }
     }
+
+    if (auto linalgOp = dyn_cast<linalg::LinalgOp>(user)) {
+      // The following check ensures that if any constants are used as an init
+      // tensor with a linalg operation, they are not cloned into the
+      // region. The rationale is the init tensor implements Destination
+      // Style Passing and so it implies that the tensor is written like a
+      // buffer.Therefore, if we have a constant within the region, then
+      // verification fails as the operation writes to a constant, immutable
+      // value. However, if the constant is kept outside the region, the access
+      // is converted to a load which is treated as a mutable value.
+      auto InitTensors = linalgOp.getDpsInits();
+      return v.getDefiningOp() && isa<arith::ConstantOp>(v.getDefiningOp()) &&
```

Comment:
Why not `m_Constant`? https://github.com/llvm/llvm-project/blob/104f5d1ff84613542442b71bcb59e19c5ed17b89/llvm/include/llvm/IR/PatternMatch.h#L848 There is also a version that does not capture the Value.
`v` is not a nice variable name